### ISSUE TITLE
[ Task x ] Make Database Helper function for Conversation 

### ISSUE THREAD

**Issue by yiwxng** [CSC207-2023Y-UofT/course-project-tindar#96]
Need to take a more careful look at current database, and helper implementation. Determine methods that would implement database helper/ creation of tables for conversation, and its relationship to Messages.

**Comment by navarhontes** [comment]
I thought we'd agreed that I'm doing this?

**Comment by yiwxng** [comment]
> I thought we'd agreed that I'm doing this?
Yes, if I am remembering this correctly, I think I made this issue before our discussion. So I think I will just reassign it to you.
